modules/mod_ginger_rdf/tests/m_rdf_tests.erl

```diff
@@ -0,0 +1,98 @@
+-module(m_rdf_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+-include("zotonic.hrl").
+-include("../include/rdf.hrl").
+
+m_find_value_test() ->
+    Input = #rdf_resource{
+        id = <<"http://dinges.com/123">>,
+        triples = [
+            #triple{
+                subject = <<"http://dinges.com/123">>,
+                predicate = <<"dc:author">>,
+                object = [
+                    #rdf_value{
+                        value = <<"Author 1">>
+                    },
+                    #rdf_value{
+                        value = <<"Author 2">>
+                    }
+                ]
```

Comment:
This is not correct: currently a triple’s object is defined as

```erlang
object :: binary() | #rdf_value{},
```

so either a single URI (binary) or a single `#rdf_value{}`, but never a list.